content/docs/concepts/security/index.md

```diff
@@ -132,7 +132,7 @@ Currently we use different certificate key provisioning mechanisms for each scen
 
 ### on-premises machines scenario
 
-1. Citadel creates a gRPC service to take CSR requests.
+1. Citadel creates a gRPC service to take [CSRs](https://en.wikipedia.org/wiki/Certificate_signing_request) (Certificate Signing Requests).
```

Comment:
Should be the opposite ordering:

(Certificate Signing Requests)[...](CSRs)

Reply:
Done.